cms/static/sass/_lib.scss

```diff
@@ -0,0 +1,9 @@
+// ------------------------------
+// Open edX Certificates: Main Style Compile
+
+// About: third party libraries and dependencies import
+
+
+@import '../../../node_modules/edx-pattern-library/node_modules/bourbon/app/assets/stylesheets/bourbon';
```

Comment:
I'm not sure that I understand @talbs's comment (maybe he'll kindly 'splain to me here... :))

> 1. We don't know where an individual app will store its npm-installed libraries, which includes any libs pulled in as UXPL dependencies. So a path made in the UXPL assets wouldn't be able to account for that until we can pass variables into an @import path (see sass/sass#279).

edx-platform consumes UXPL. UXPL knows how npm is configured, hence the local path from the UXPL project root to its `node_modules` directory, and therefore `_rtl.scss` can safely encapsulate that. edx-platform knows _its_ npm configuration, hence it knows where `_rtl.scss` is installed. I am genuinely confused as to what's preventing us from importing the `_rtl.scss` (or any other) partial.

> 2. The `$layout-direction: ltr;`, `$grid-direction-default: ltr;`, and `$grid-direction-reversed: ltr;` variables need to be set specifically for an app's main `*-ltr.scss` and `*--rtl.scss` compile files. I had originally had this logic in the UXPL using the `@if` + `@import` method in the UXPL that we removed recently (edx/ux-pattern-library#263).

It makes total sense to me for us to define our own SASS variables in this way.